alkira/resource_alkira_service_checkpoint.go

```diff
@@ -162,12 +183,12 @@ func resourceAlkiraCheckpoint() *schema.Resource {
 			"segment_ids": {
 				Description: "The IDs of the segments associated with the service.",
 				Type:        schema.TypeList,
-				Required:    true,
+				Optional:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
 			},
 			"segment_options": {
 				Type:        schema.TypeSet,
-				Optional:    true,
+				Required:    true,
```

Comment:
Is this right? segment_id is optional? I think it's always required, but `segment_options` could be optional.

Reply:
When I would try to create without segment_options it would complain that segment_options cannot be nil.